superset-frontend/packages/superset-ui-demo/storybook/stories/plugins/plugin-chart-echarts/Timeseries/Stories.tsx

```diff
@@ -87,3 +88,31 @@ export const Timeseries = ({ width, height }) => {
     />
   );
 };
+
+export const WithNegativeNumbers = ({ width, height }) => (
+  <SuperChart
+    chartType="echarts-timeseries"
+    width={width}
+    height={height}
+    queriesData={[{ data: negativeNumData }]}
```

Comment:
I noticed that the storybook isn't formatting the temporal x-axis correctly:
![image](https://user-images.githubusercontent.com/33317356/184075959-280fd0ad-58bf-4797-baba-f27ded1e5817.png)
Could we add an entry for `__timestamp` to `colnames` and `coltypes` which would normally be returned in the chart data payload so the formatting looks correct (2 comes from `GenericDataType.TEMPORAL`)? I believe this should fix it:

```suggestion
    queriesData={[{ data: negativeNumData, colnames: ['__timestamp'], coltypes: [2] }]}
```

Reply:
Thanks for hotfix! I'll update accordingly